app/Audit/AuditLogFormatterFactory.php

```diff
@@ -75,4 +84,43 @@ public function make(AuditContext $ctx, $subject, $eventType): ?IAuditLogFormatt
         $formatter->setContext($ctx);
         return $formatter;
     }
+
+    private function getFormatterByContext(object $subject, string $event_type, AuditContext $ctx): ?IAuditLogFormatter
+    {
+        $class = get_class($subject);
+        $entity_config = $this->config['entities'][$class] ?? null;
+        
+        if (!$entity_config || !isset($entity_config['strategies'])) {
+            return null;
+        }
+
+        foreach ($entity_config['strategies'] as $strategy) {
+            if (!$this->matchesStrategy($strategy, $ctx)) {
+                continue;
+            }
+
+            $formatter_class = $strategy['formatter'] ?? null;
+            return $formatter_class ? new $formatter_class($event_type) : null;
+        }
+
+        return null;
+    }
+
+    private function matchesStrategy(array $strategy, AuditContext $ctx): bool
+    {
+        if (isset($strategy['route']) && !$this->routeMatches($strategy['route'], $ctx->route)) {
+            return false;
+        }
+
+        return true;
+    }
+
+    private function routeMatches(string $pattern, string $actual_route): bool
+    {
+        $normalized_pattern = preg_replace('/\{[a-zA-Z_]+\}/', '\d+', $pattern);
```

Comment:
@andrestejerina97 we dont need this lets just add the rawRoute ( without the bindings at AuditContext)
here
https://github.com/OpenStackweb/summit-api/blob/ee9422d527e501b87c0234a30193b55495d7d30b/app/Audit/AuditEventListener.php#L111

like this

rawRoute: Route::getRoutes()->match($req);
and then it just a simple string comparison